client-src/elements/chromedash-feature-detail.js

```diff
@@ -320,7 +320,7 @@ class ChromedashFeatureDetail extends LitElement {
   }
 
   renderGateChips(feStage) {
-    const gatesForStage = this.gates.filter(g => g.stage_id == feStage.id);
+    const gatesForStage = this.gates.filter(g => g.stage_id == feStage.stage_id);
```

Comment:
This is a redundant property name, which is my fault. We can look into changing this at some point moving forward.